features/scv/scvcli/src/main/java/org/opennms/features/scv/cli/ScvCli.java

```diff
@@ -78,6 +78,20 @@ public SecureCredentialsVault getSecureCredentialsVault() {
         return secureCredentialsVault;
     }
 
+    private static String lookupKeyStore() {
+        Properties properties = new Properties();
+        String defaultKeyStore="scv.jce";
+        try {
+            properties.load(ScvCli.class.getResourceAsStream("/scvcli-filtered.properties"));
+            String opennmsHome = properties.getProperty("install.dir");
+            SecureCredentialsVault.loadScvProperties(opennmsHome);
```

Comment:
You are loading system properties but defaultKeyStore wasn't updated

Reply:
I've no idea what you are doing here? You do not alter defaultKeyStore, right?